python/neuroglancer/pipeline/storage.py

```diff
@@ -214,30 +342,34 @@ def get_path_to_file(self, file_path):
                              file_path])
         return  os.path.join(*clean)
 
-    def put_file(self, file_path, content, compress):
+    def put_file(self, file_path, content, content_type, compress):
```

Comment:
Another option is to create a second file 
with .metadata extension and ignore those during listing